docs/platforms/javascript/common/opentelemetry/custom-setup.mdx

```diff
@@ -180,11 +204,14 @@ const provider = new NodeTracerProvider({
 // Validate that the setup is correct
 Sentry.validateOpenTelemetrySetup();
 ```
+
 ## ESM Loaders
 
-If your application is running in ESM (`import`/`export` syntax), OpenTelemetry requires a [special setup](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) to work correctly.
-The Sentry SDK will automatically detect if your application is running in ESM mode and by default [register a loader hook](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md#instrumentation-hook-required-for-esm) itself.
-If you are also registering a loader hook, you need to disable Sentry's loader hook:
+If your application is running in ESM (`import`/`export` syntax), OpenTelemetry requires you to set up _ESM loader hooks_.
+
+The Sentry SDK will automatically register ESM loader hooks by default.
+However, if you have your own OpenTelemetry setup, it is recommended to configure the Sentry SDK to not register these hooks and instead register them yourself.
+You can do so by setting `registerEsmLoaderHooks` to `false` and [set up ESM loader hooks](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md#instrumentation-hook-required-for-esm):
```

Comment:
I think it would be preferable to recommend the Sentry hook here that wraps the otel hook. That would leave us some options in the future!